source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -26,15 +27,31 @@ To update elements in an array, perform the following actions:
 Sample Data
 ~~~~~~~~~~~
 
+The examples in this guide use the following ``Book`` struct as a model for documents
+in the ``books`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/updateArray.go
+   :start-after: start-drink-struct
+   :end-before: end-drink-struct
+   :language: go
+   :dedent:
+
+The ``truncate`` :ref:`struct tag<golang-struct-tags>` allows the driver
+to truncate types such as ``float64`` to ``int32`` when unmarshalling.
+
 To run the examples in this guide, load the sample data into the
-``quantity.tea`` collection with the following
-snippet:
+``db.drinks`` collection with the following snippet:
 
 .. literalinclude:: /includes/fundamentals/code-snippets/CRUD/updateArray.go
    :language: go
    :dedent:
-   :start-after: begin insert docs
-   :end-before: end insert docs
+   :start-after: begin insertDocs
+   :end-before: end insertDocs
+
+Each document contains a description of a drink that
+includes the drink's description, available sizes (in ounces), and available
```

Comment:
Suggestion: I would try to avoid parenthesis if possible

```suggestion
includes the drink's description, available ounce sizes, and available
```

or

```suggestion
includes the drink's description, available sizes in ounces, and available
```